### ISSUE TITLE
Audio Conferencing complimentary dial-out period

### ISSUE THREAD

**Issue by hmank** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#116]
The complimentary-dial-out-period should be in a different section as it does not fall into the Accessibility-and-Regulatory category, it is more suited under a Life-cycle subsection to Skype for Business ie. https://docs.microsoft.com/en-us/SkypeForBusiness/Life-cycle/complimentary-dial-out-period

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fee2f094-5a08-489c-03ee-654a9f259b32
* Version Independent ID: 813cfbf3-bb73-70ab-610d-68fbf302f6bb
* Content: [Complimentary dial-out period](https://docs.microsoft.com/en-us/SkypeForBusiness/accessibility-and-regulatory/complimentary-dial-out-period)
* Content Source: [Skype/SfbOnline/accessibility-and-regulatory/complimentary-dial-out-period.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbOnline/accessibility-and-regulatory/complimentary-dial-out-period.md)
* Service: **skype-for-business-online**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by tonysmit** [comment]
I will consider renaming the parent topic. Thanks for the feedback.
